public/locales/en-US/send.ftl

```diff
@@ -50,6 +51,7 @@ downloadButtonLabel = Download
     .title = Download
 downloadNotification = Your download has completed.
 downloadFinish = Download Complete
+fileSizeProgress = ({ $partialSize } of { $totalSize })
```

Comment:
This PR adds a couple strings for l10n, but doesn't use them yet. @flodolo, is it a good idea to pre-stage strings so they can be translated before using them, or better just to use them now? 

In `fileSizeProgress` case it's a new string previously not localized, whereas `uploadingPageProgress` will replace the `uploadingPageHeader` to be symmetric with `downloadPageProgress`.